src/js/logic.js

```diff
@@ -0,0 +1,33 @@
+import { storyObj } from "./index";
+import { render } from "./render";
+import { createChapterStructure } from "./chapterDOM";
+import { createChapterEnd } from "./chapterEnd";
+
+const gameLogic = (() => {
+  //This function is our callback for drag and drop.
+  const displayChapterEnd = (chapter) => {
+    const chapterNumber = chapter.getChapterNumber();
+    const quiz = "Quiz";
+    const nextChapter = goToNextChapter(chapter).setupChapterPage();
+    const chapterEnd = createChapterEnd(chapterNumber, quiz, nextChapter);
+
+    render(chapterEnd);
+  };
+
+  const goToNextChapter = (chapter) => {
+    chapter.setCompletitionStatus();
```

Comment:
Could you correct the typo in setCompletitionStatus() already in your PR in the files where it happens?

Reply:
Sure